pyracf/common/add_operation_error.py

```diff
@@ -0,0 +1,24 @@
+"""Exception to use when Add operation would alter an existing profile."""
+
+
+class AddOperationError(Exception):
+    """
+    Raised when a profile passed into an Add is successfully extracted.
+    """
+
+    def __init__(self, profile_name: str, class_name: str) -> None:
```

Comment:
I suggest changing all values for `class_name` and `admin_types` to camel case (i.e., `user`, `group`, `dataSet`). Also note that when raising the exception, the class name can be passed to the exception using the `SecurityAdmin.__profile_type` class attribute by making it into a protected attribute (`SecurityAdmin._profile_type`) so that it can be used in the child classes for the exception handlers.

Reply:
Agree, that's cleaner. Change made across the board. Corresponding changes also made in unit testing where the error contents are evaluated.